File: 3274. Check if Two Chessboard Squares Have the Same Color.cpp
class Solution
{
public:
    bool checkTwoChessboards(string coordinate1, string coordinate2)
    {
        int one = coordinate1[0] + coordinate1[1];
        int two = coordinate2[0] + coordinate2[1];

        return one % 2 == two % 2;
    }
};

File: 3274. Check if Two Chessboard Squares Have the Same Color.py
class Solution(object):
    def checkTwoChessboards(self, coordinate1, coordinate2):
        # Convert coordinates to ASCII and add their values
        one = ord(coordinate1[0]) + int(coordinate1[1])
        two = ord(coordinate2[0]) + int(coordinate2[1])

        # Return true if both sums have the same parity
        return one % 2 == two % 2